src/app/components/videochat/jh-video-chat.html

```diff
@@ -1,65 +1,137 @@
+        </button>
+      </div>
+      <!-- Hotkey Button -->
+      <div class="sidebar-box">
         <jh-pushtotalk-button></jh-pushtotalk-button>
+      </div>
+      <!-- Audio Button -->
+      <div class="sidebar-box">
         <jh-audio-button feed="vm.mainFeed()"></jh-audio-button>
       </div>
-    </div>
-  </header>
-</div>
+      <!-- Video Button -->
+      <div class="sidebar-box">
+        <jh-video-button feed="vm.mainFeed()"></jh-video-button>
+      </div>
+      <!-- Window Share Button -->
+      <div class="sidebar-box">
+        <button
+           class="btn btn-sidebar"
+           <span><img class="btn-img" src="images/buttons/buttons_window-share.svg"/></span>
+        </button>
+      </div>
+      <!-- Screen Share Button -->
+      <div class="sidebar-box">
+        <button
+           class="btn btn-sidebar"
+           <span><img class="btn-img" src="images/buttons/buttons_screen-share.svg"/></span>
+        </button>
+      </div>
+    </ul>
+  </div>
 
-<div class="row" id="videochat-body">
-  <div id="videochat-playroom">
-    <ul>
-      <li activity-item
-          row="item.position[0]" col="item.position[1]"
-          size-x="item.size[0]" size-y="item.size[1]"
-          ng-repeat="item in vm.activityItems">
-        <div
-          ng-if="item.content === 'Speaker'"
-          class="grid-item-content"
-          id="main-video"
-          jh-main-feed="true"
-          feed="vm.highlightedFeed()"
-          ng-class="{mirrored: vm.mirrored()}">
+  <!-- Header -->
+  <div id="header-wrapper">
+    <header>
+      <div class="row">
+        <!-- Menu Toggle -->
+        <div class="col-xs-5 hidden-sm hidden-md hidden-lg text-left">
+          <button href="#menu-toggle" id="menu-toggle">
+            <span class="glyphicon glyphicon-menu-hamburger"></span>
+          </button>
         </div>
-        <div
-          ng-if="item.content === 'Participants'"
-          class="grid-item-content"
-          id="thumbnails">
-          <div
-            jh-feed="true"
-            feed="f"
-            ng-repeat="f in vm.feeds() | orderBy:'+display'"
-            toggle-highlight-fn="vm.toggleHighlightedFeed(f)"
-            highlighted="vm.isHighlighted(f)"
-            highlighted-by-user="vm.isHighlightedByUser(f)">
-          </div>
+        <!-- Logo + Footer -->
+        <div class="hidden-xs col-sm-10 text-left">
+          <jh-footer></jh-footer>
         </div>
-        <div
-          ng-if="item.content === 'Chat'"
-          class="grid-item-content">
-          <jh-chat messages="vm.logEntries()"></jh-chat>
+        <!-- Username + Room Name + Logout -->
+        <div class="col-xs-19 col-sm-14 text-right">
+          <jh-room-info></jh-room-info>
+          <jh-main-feed-info></jh-main-feed-info>
+          <jh-leave-button></jh-leave-button>
         </div>
-      </li>
-    </ul>
+      </div>
+    </header>
   </div>
+
+  <!-- Page Content -->
```

Comment:
I only commented it out because it was easier to work with without all that noise there. Would you like me to uncomment it?